plugins/modules/aci_l4l7_concrete_device.py

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_l4l7_concrete_device
+short_description: Manage L4-L7 Concrete Devices (vns:CDev)
+description:
+- Manage L4-L7 Concrete Devices.
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  device:
+    description:
+    - Name of the logical device (vns:lDevVip) the concrete device is attached to
```

Comment:
add reference to the M(ansible.aci.aci_l4l7_device) module

Reply:
added